sv_pipeline/exome/load_data.py

```diff
@@ -273,14 +306,14 @@ def _parse_row(row, header_indices):
         if sample_remap and sample_id in sample_remap:
             sample_id = sample_remap[sample_id]
         if sample_subset is None or sample_id in sample_subset:
-            parse_sv_row(row, parsed_svs_by_name, header_indices, sample_id)
+            parse_sv_row(row, parsed_svs_by_name, header_indices, sample_id, is_new_joint_call)
             found_samples.add(sample_id)
             return True
         else:
             skipped_samples.add(sample_id)
             return False
 
-    load_file(input_dataset, _parse_row, out_file_path=out_file_path)
+    load_file(input_dataset, _parse_row, out_file_path=out_file_path, columns=None if is_new_joint_call else MERGED_CALL_COLUMNS)
```

Comment:
the `columns=None` as the way of handling the default is confusing, I think this would be better if you made columns required and explicitly set it in both cases